### PR TITLE
Add testnet label to faucet transactions and update faucet address

### LINKED ISSUE #485: Faucet transactions should be labeled and have the Celo avatar
### Expected Behavior

Transaction from the faucet should be labeled "{Testnet Name} Faucet".  Should also have the Celo avatar (rather than #)

### Current Behavior

Transactions from the faucet show the # avatar.

Screenshot_2019-08-14-11-17-07_censored

### PR DESCRIPTION
**Pull request by drklee3**
### Description

Updates the faucet address to point to the correct address on alfajores ([`0xcea3ef8e187490a9d85a1849d98412e5d27d1bb3`](https://alfajores-blockscout.celo-testnet.org/address/0xcea3ef8e187490a9d85a1849d98412e5d27d1bb3/transactions)).  Also uses `Celo (TESTNET) Faucet` as the transaction name instead of just `Received`.

| Before | After |
| ------ | ----- |
|  Screenshot_2019-08-14-11-17-07_censored |  Screenshot_2019-08-14-15-53-02_censored |

### Tested

Tested the mobile app using a local `blockchain-api` instance with alfajores environment variables (BlockScout, Celo Gold/Dollar/Faucet addresses) to verify if the faucet transaction displays properly.  

### Other changes

N/A

### Related issues

- Fixes #485

### CODE REVIEW

File: packages/mobile/src/transactions/TransferFeedItem.tsx [hunk 1]
```diff
@@ -169,7 +170,10 @@ export class TransferFeedItem extends React.PureComponent<Props> {
       comment = null
     } else if (type === TransactionTypes.FAUCET) {
       contactImage = <Image source={faucetIcon} style={styles.image} />
-      fullName = 'Celo'
+      const testnetName = DEFAULT_TESTNET
```
Comment (jmrossy) [inline packages/mobile/src/transactions/TransferFeedItem.tsx]: let's use lodash for this, would be neater: https://stackoverflow.com/questions/38084396/lodash-title-case-uppercase-first-letter-of-every-word

Comment (drklee3) [reply packages/mobile/src/transactions/TransferFeedItem.tsx]: Ah right there is lodash.

File: packages/blockchain-api/app.alfajores.yaml [hunk 2]
```diff
@@ -7,6 +7,6 @@ env_variables:
   # Pull addresses from the build artifacts of the network in protocol/build
   CELO_GOLD_ADDRESS: '0x4813BFD311E132ade22c70dFf7e5DB045d26D070'
   CELO_DOLLAR_ADDRESS: '0x299E74bdCD90d4E10f7957EF074ceE32d7e9089a'
-  FAUCET_ADDRESS: '0x456f41406B32c45D59E539e4BBA3D7898c3584dA '
+  FAUCET_ADDRESS: '0xcea3ef8e187490a9d85a1849d98412e5d27d1bb3'
```
Comment (jmrossy) [inline packages/blockchain-api/app.alfajores.yaml]: FYI this may change again today after we re-deploy. Check with @timmoreton or @asaj

**Review by jmrossy — DISMISSED** [review]

**Review by annakaz — APPROVED** [review]
LGTM!

**Pull request merged**